src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -195,7 +194,7 @@ class ValidateCodeForm extends React.Component {
                 ) : (
                     <View style={[styles.mv3]}>
                         <TextInput
-                            autoComplete={Platform.select({android: 'sms-otp', web: 'one-time-code', ios: 'one-time-code'})}
+                            autoComplete={this.props.autoComplete}
```

Comment:
@narefyev91 I think we should add the autocomplete prop to the PropTypes and add it oneOf type where you will list the two options we have

Reply:
Sure - let me add this now